src/photos/lib/onPhotoTrashed.js

```diff
@@ -0,0 +1,42 @@
+import { getReferencedBy } from 'cozy-client'
```

Comment:
no cozy-client update?

Reply:
If, of course, the CI turns overnight. I updated it this morning